docs/components/button-menu.md

```diff
@@ -1,37 +1,140 @@
+- for items in a [GOV.UK Design System summary list](https://design-system.service.gov.uk/components/summary-list/) - add link text to each row
+- to display options like on a [GOV.UK Design System question page](https://design-system.service.gov.uk/patterns/question-pages/)  - list them as bullets
+- for warning or inactive (disabled) buttons
+
+### Things to consider
+
+When deciding on the menu length consider that:
+
+- without JavaScript, all the menu items will appear as a row
+- very long button menus can be a sign that the page needs to be simplified or the journey split over multiple pages
+
+### Similar or linked components
+
+There’s also the:
+
+- [button component on the GOV.UK Design System](https://design-system.service.gov.uk/components/button/)
+- [page header actions component](https://design-patterns.service.justice.gov.uk/components/page-header-actions/)
+- [summary list component on the GOV.UK Design System](https://design-system.service.gov.uk/components/summary-list/)
+- [identity bar component](https://design-patterns.service.justice.gov.uk/components/identity-bar/)
 
 ## How to use
 
-By default this component just groups buttons together to ensure they are spaced correctly. But can be turned into a toggle menu at a configurable screen width that:
+### Position on the page
+
+Place the button menu near the heading that the tasks relate to. To add it next to a heading, put it in the same container. To add it to an H1 use the [page header actions component](https://design-patterns.service.justice.gov.uk/components/page-header-actions/).
+
+For tabbed content, adding it within the tabs may help users to understand:
+
+- what the menu items relate to
+- if (and how) the menu items change when the user moves between tabs
+
+You can place it on the left or right. Match the alignment of the buttons with the menu: place them on the left of a left-aligned menu, and on the right of a right-aligned menu.
+
+Screen magnification users may not find a right-aligned menu if:
+
+- content is spread all over the page
+- there’s a lot of whitespace around the menu
+- the button menu is a long way from the heading
+
+Placing the menu on the right may stop it from obscuring other items on the screen, such as a case list or side navigation.
+
+### Left-aligned menu items
+
+{% example "/examples/button-menu-left-aligned", 275 %}
+
+### Right-aligned menu items
+
+{% example "/examples/button-menu-right-aligned", 275 %}
+
+### Grouping buttons
+
+You can add a button menu alongside a link or GOV.UK button. This code has correct spacing and creates a better user experience for people on mobile devices.
+
+{% example "/examples/button-menu-right-aligned", 275 %}
+
+
+### Button colour
+
+All the menu items are grey. The colour of the menu title button follows the [GOV.UK Design System guidance on button colours](https://design-system.service.gov.uk/components/button/#default-buttons). It defaults to green (default) but you can change it.
```

Comment:
This is a case of docs and code getting out of sync - it was like this, then I thought I worked out how to default to grey, but I've just now reverted to defaulting to green.

The reason being that to override it you had to pass a class of "govuk-button" (or any other class) but assuming you didn't need a custom class, you'd have to pass the "govuk-button" class to prevent the "secondary" class being applied, which felt very un-intuitive!